lib/index.ts

```diff
@@ -66,6 +70,14 @@ export async function inspect(
       .then(createPackageTree);
   }
 
+  if (options['target-framework'] && !options['dotnet-runtime-resolution']) {
+    return Promise.reject(
+      new CliCommandError(
+        'target framework flag is currently only supported when also scanning with runtime resolution using the `--dotnet-runtime-resolution` flag',
```

Comment:
Is it ever going to be supported without `--dotnet-runtime-resolution`?

Reply:
Yes, my current hope is that as .NET frameworks become outdated, we at some point drop support for everything below the versions that this logic supports, drop that flag and make it work as default behavior.